package.json

```diff
@@ -28,5 +28,11 @@
     "nuxt": "^2.15.3",
     "siroc": "^0.10.1",
     "standard-version": "^9.1.1"
+  },
+  "peerDependencies": {
+    "browserslist": "^4.0.0"
```

Comment:
Using peerDependency means end-user or parent package should install this dependency but seems consumer is not our module? In this case maybe directly adding browserlist to dependencies?